### ISSUE TITLE
Accessible close button

### ISSUE THREAD

**Issue by Nettsentrisk** [burocratik/outdated-browser#255]
The close button is now solved with an empty anchor link:
`<a href="#" id="btnCloseUpdateBrowser" title="Close">&times;</a>`

This has poor accessibility and should be updated to this pattern across the project:
`<button id="btnCloseUpdateBrowser" title="Close" aria-label="Close">&times;</button>`

**Comment by brunoamorim** [comment]
@Kum0ri and @Deadload what do you think about that?

**Comment by alexandre-works** [comment]
Hi guys,
I am not 100% sure, but I think the reason behind the <a> and not <button> is due to IE6 and IE7, specially on IE6 there where some problems with this tag. Best to test it specially on IE6, if all is well I think os a good idea to change it.
Cheers

**Comment by Nettsentrisk** [comment]
I should have specified that it should be button[type=button]. This shouldn't cause any problems in older IE versions when Javascript is being used to handle the onclick event anyway.

The only issue I can find with older IE and button elements is it posting the values of all buttons in a form regardless of whether they were pressed or not. That's not an applicable issue for this scenario.

**Comment by branditodesigns** [comment]
I thought &times; is read out as multiplied or multiplication symbol on screen readers, so that won't work well either.

**Comment by Nettsentrisk** [comment]
aria-label overwrites the content of the node, so the multiplication symbol would never be read by the screen reader if the changes I suggested in OP were implemented.

**Comment by vandijkstef** [comment]
While semantically correct, the usage of `button` did make it harder to work with already used CSS in my case. Also, the fallback stated in JS is still using `a` over a `button`. *Inconsistency detected*
